Source/DafnyDriver/Commands/RunCommand.cs

```diff
@@ -49,7 +49,7 @@ static RunCommand() {
       Concat(DafnyCommands.ConsoleOutputOptions).
       Concat(DafnyCommands.ResolverOptions);
 
-  public static Command Create() {
+  public static Command Create(Func<DafnyOptions, int> continuation = null) {
```

Comment:
What is this for?

Reply:
It's to be able to create a Run command but with a continuation different than the one that launches the regular pipeline, to be able to test the parsing of options only. If you have any other way I could achieve the same result, I'd happily consider it.